dev-libs/libqtxdg/libqtxdg-3.8.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Are you able to do EAPI 8 now (do the eclasses allow it?)?

Reply:
I will check! Can I simply bump the EAPI and attempt an emerge to test and see if there is any failure or warning?